arch/configure.defaults

```diff
@@ -2068,6 +2068,49 @@ NETCDFPAR_BUILD	=      CONFIGURE_NETCDFPAR_BUILD
 
 #insert new stanza here
 
+###########################################################
+#ARCH    Linux aarch64, NVHPC compiler with nvc # serial smpar dmpar dm+sm
```

Comment:
It looks to me like this would for `x86_64` as well as for `aarch64`. If you agree, we could change this line to something like:
```
#ARCH    Linux aarch64 x86_64, NVHPC compiler with nvc # serial smpar dmpar dm+sm
```

Reply:
That's correct - today! But there may be aarch64 specific flags we would like to add in the future so my preference would be to keep them separate.

Reply:
That sounds reasonable -- thanks for clarifying.